ui/core/src/actions/clp/index.ts

```diff
@@ -77,7 +77,7 @@ export default ({
   }
 
   const actions = {
-    async swap(sentAmount: AssetAmount, receivedAsset: Asset) {
+    async swap(sentAmount: AssetAmount, receivedAsset: AssetAmount) {
```

Comment:
Would it be better to call this `minReceivedAmount:AssetAmount` then it is clearer as to the intention here?